src/SwiftUI/SwiftUI/Color.cs

```diff
@@ -75,6 +94,35 @@ public Color (double hue, double saturation, double brightness, double opacity)
 		{
 			Data = CreateFromHSBO (hue, saturation, brightness, opacity);
 		}
+
+		public Color (RGBColorSpace colorSpace, double red, double green, double blue, double opacity)
+		{
+			Data = CreateFromRGBColorSpaceRedGreenBlueOpacity (GetSwiftUIColorSpace (colorSpace), red, green, blue, opacity);
+		}
+
+		public Color (RGBColorSpace colorSpace, double white, double opacity)
+		{
+			Data = CreateFromRGBColorSpaceWhiteOpacity (GetSwiftUIColorSpace (colorSpace), white, opacity);
+		}
+
+		IntPtr GetSwiftUIColorSpace (RGBColorSpace colorSpace)
```

Comment:
This method looks like it could be static, and it could also use one of those fancy new switch expressions

Reply:
Agreed! Not used the new fan dangled version yet, but this looks pretty sexy
			```
return colorSpace switch
			{
				RGBColorSpace.DisplayP3 => ColorSpaceDisplayP3,
				RGBColorSpace.sRGB => ColorSpacesRGB,
				RGBColorSpace.sRGBLinear => ColorSpacesRGBLinear,
				_ => IntPtr.Zero
			};
```